chromadb/db/index/hnswlib.py

```diff
@@ -71,7 +75,7 @@ class Hnswlib(Index):
     _index_metadata: IndexMetadata
     _params: HnswParams
     _id_to_label: Dict[str, int]
-    _label_to_id: Dict[int, str]
+    _label_to_id: Dict[int, UUID]
```

Comment:
See below comment on how its odd that we store the string and the uuid in opposite direction. Updating the typing.